src/variorum/config_architecture.h

```diff
@@ -66,6 +66,20 @@ enum ibm_arch_e
     POWER9 = 9, // Power9
 };
 
+/// @brief List of NVIDIA family and models.
+/// @todo: we need to find architecture detection info here.
+enum nvidia_arch_e
+{
+    VOLTA = 1, // Volta
+};
+
+/// @brief Mapping topology information.
+struct map
+{
+    /// @brief Unique ID of each physical core.
+    int physical_core_idx;
+};
+
```

Comment:
Do you still need this struct after the recent hwloc change?

Reply:
Not anymore. Removed the 'map' structure as the hwloc topology can be used now.